Day_26_27/miracleDay_26_27.scilla

```diff
@@ -0,0 +1,139 @@
+scilla_version 0
+
+import BoolUtils
+
+library MyContractLib
+
+
+(********* Part -5 Global Assignment "A" *************)
+
+type Question=
+  | Question of String ByStr20
+
+let one_msg =
+fun (msg : Message) =>
+let nil_msg = Nil {Message} in
+  Cons {Message} msg nil_msg
+
+let reward = Uint128 10000000000000 (reward 10 Zil)
```

Comment:
**(reward 10 Zil)** is invalid.
Please properly comment it.

Reply:
> 
> 
> **(reward 10 Zil)** is invalid.
> Please properly comment it.

properly commented now